feat: add continuous profiling nodejs section (#11186)
getsentry/sentry-docs: docs/platforms/javascript/guides/node/profiling/index.mdx

);
```

## Enable Continuous Profiling

<Include name="feature-stage-beta.mdx" />

_(New in version @TODO)_

> The current profiling implementation stops the profiler automatically after 30 seconds (unless you manually stop it earlier). Naturally, this limitation makes it difficult to get full coverage of your app's execution. We now offer an experimental continuous mode, where profiling data is periodically uploaded while running, with no limit to how long the profiler may run.

Reviewer: ```suggestion
The current profiling implementation stops the profiler automatically after 30 seconds (unless you manually stop it earlier). Naturally, this limitation makes it difficult to get full coverage of your app's execution. We now offer an experimental continuous mode, where profiling data is periodically uploaded while running, with no limit on how long the profiler may run.
```

To get started with continuous profiling, you can start and stop the profiler directly with `Sentry.profiler.startProfiling` and `Sentry.profiler.stopProfiling`.

<Note>

> If you had previously set `profilesSampleRate` or `profilesSampler` to use transaction-based profiling, then remove those lines of code from your configuration.

Reviewer: ```suggestion
If you previously set `profilesSampleRate` or `profilesSampler` to use transaction-based profiling, you must remove those lines of code from your configuration in order to use continuous profiling.
```

</Note>

```js
const Sentry = require('@sentry/node');

Sentry.init({
    dsn: "___PUBLIC_DSN___" ,
})

Sentry.profiler.startProfiling();

// run some code here

Sentry.profiler.stopProfiling();
```

> These new APIs do not offer any sampling functionality—every call to start the profiler. If you are interested in reducing the amount of profiles that run, you must take care to do it at the callsites.

Reviewer: ```suggestion
These new APIs do not offer any sampling functionality—every call to start the profiler will run and start sending profiling data. If you are interested in reducing the amount of profiles that run, you must take care to do it at the callsites.
```
